NetMQ.ReactiveExtensions/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {    
+    "NetMQ": "4.0.0-rc5",
+    "protobuf-net": "2.1.0",
+    "System.Reactive.Core": "3.1.0"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.1"
+        }
+      },
+      "imports": "dnxcore50"
+    },
+    "netstandard1.6": {
+      "dependencies": {
+        "NETStandard.Library": "1.6.0"
+      }
+    },
+    "net45": {
+      "dependencies": {
+        "System.Xml.Linq" : "*" 
```

Comment:
Ok, I've added it to my todo list. It's 10pm and I'm quite tired now so I'll turn in for the night. Very happy to see that everything is working nicely!